action.yml

```diff
@@ -57,5 +57,5 @@ outputs:
   changes:
     description: Total number of changes
 runs:
-  using: node16
+  using: node20
```

Comment:
How do we test this? do we have CI to make sure the Action works fine?

Reply:
We run it here and all works just fine:
https://github.com/kamilkisiela/graphql-inspector/actions/runs/9112064261/job/25050545293